components/form/TextField.tsx

```diff
@@ -7,22 +7,38 @@ import {
   Text,
   Input,
   InputProps,
+  InputGroup,
+  InputLeftElement,
 } from '@chakra-ui/react';
 
 type Props = InputProps & {
   readonly name: string;
-  readonly label: string;
+  readonly label: string | ReactNode;
```

Comment:
Hm I came across the same problem with the label and I added an extra prop for a `labelHelper`, it looks like yours is more robust and a bit cleaner so i'll go with this too.